include/dlaf/eigensolver/gen_to_std.h

```diff
@@ -38,6 +38,7 @@ namespace eigensolver {
 /// Note: B should be modifiable as the diagonal tiles might be temporarly modified during the calculation.
 /// @pre mat_a and mat_b have the same square size,
 /// @pre mat_a and mat_b have the same square block size,
+/// @pre mat_a and mat_b have the same tile tile and block sizes,
```

Comment:
I have seen it a couple of times over this PR 😉 

```suggestion
/// @pre mat_a and mat_b have the same tile and block sizes,
```

Reply:
https://github.com/eth-cscs/DLA-Future/pull/935/commits/9c50411e7d5a942bee2953563b1e0800142ec054